zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/publishing/PostPublisher.java

```diff
@@ -105,14 +106,13 @@ public static boolean postPublish(Zebedee zebedee, Collection collection, boolea
 
             collection.delete();
             ContentTree.dropCache();
+            zebedee.getSchedulerKeyCache().remove(collection.getId());
 
-            // FIXME using PostPublisher.getPublishedCollection feels a bit hacky
             SlackNotification.publishNotification(publishedCollection,SlackNotification.CollectionStage.POST_PUBLISH, SlackNotification.StageStatus.COMPLETED);
 
             return true;
         } catch (Exception exception) {
-            error().data("collectionId", collection.getDescription().getId()).logException(exception, "An error occurred during the publish cleanup");
-            // FIXME using PostPublisher.getPublishedCollection feels a bit hacky
+            error().collectionID(collection).exception(exception).log("An error occurred during the publish cleanup");
```

Comment:
To be specific in logs, should we not capture collectionID for every exception handling even for null

Reply:
ignoring this comment in assumption of traceID in logs are out of scope of this PR.